plux/build/setuptools.py

```diff
@@ -252,7 +294,11 @@ def _is_local_build_context():
         try:
             i = sys.argv.index("--egg-base")
         except ValueError:
-            return False
+            try:
+                # this code path is for building wheels from source distributions via pip
+                i = sys.argv.index("--output-dir")
```

Comment:
🤯

Reply:
it's not great we're depending on assumptions how pip calls setuptools, but :shrug: